source/components/molecules/NavigationButtonField/NavigationButtonField.tsx

```diff
@@ -66,7 +66,7 @@ const NavigationButtonField: React.FC<Props> = ({
     <ButtonFieldWrapper>
       <Button variant="outlined" onClick={onClick} colorSchema={colorSchema}>
         {iconName.length ? <Icon name={iconName} /> : null}
-        {text && <Text>{text}</Text>}
+        {text ? <Text>{text}</Text> : null}
```

Comment:
What is the reason for returning `null` ?

Reply:
Missed restoring it after debugging